Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.cpp

```diff
@@ -1497,7 +1534,11 @@ struct SymbolDictionaryDecodingInputParameters {
     u32 number_of_new_symbols { 0 };      // "SDNUMNEWSYMS" in spec.
     u32 number_of_exported_symbols { 0 }; // "SDNUMEXSYMS" in spec.
 
-    // FIXME: SDHUFFDH, SDHUFFDW, SDHUFFBMSIZE, SDHUFFAGGINST
+    // Only set if uses_huffman_encoding is true.
+    JBIG2::HuffmanTable* delta_height_table;               // "SDHUFFDH" in spec.
+    JBIG2::HuffmanTable* delta_width_table;                // "SDHUFFDW" in spec.
+    JBIG2::HuffmanTable* bitmap_size_table;                // "SDHUFFBMSIZE" in spec.
+    JBIG2::HuffmanTable* number_of_symbol_instances_table; // "SDHUFFAGGINST" in spec.
```

Comment:
What's the ownership model of these?
are they guaranteed to be a standard table, or am I missing something
(Theres `MaybeOwned` in any case)

Reply:
Owned by the caller. The function doesn't store references to it, and the parameters are only created to be passed in.